### DISCUSSION TITLE
AssertionError #1 Config Auth.userProvider is not a valid UserProvider.

### DISCUSSION THREAD

**Discussion by lavahasif** [codeigniter4/shield#543] [Q&A] [answered]
AssertionError #1
Config Auth.userProvider is not a valid UserProvider. 

showing like this while user registration.what i wrong it ?

**Comment by kenjis** [comment, marked as answer]
Check the `Config\Auth::$userProvider` value.
https://github.com/codeigniter4/shield/blob/develop/docs/concepts.md#user-providers

**Reply by lavahasif** [reply]
Thank you for your valuable answer .I didnt use myown user model.i used defualt shield usermodel.I just reinstall the package and .re rerun the spark migrate .it works.

**Reply by lavahasif** [reply]
![image](https://user-images.githubusercontent.com/22430922/203980741-115965a6-f36e-4ca5-b161-303ed2f8c8d8.png)
both prvider works
![image](https://user-images.githubusercontent.com/22430922/203981472-cc86fb95-2275-4cbe-9fd0-3e919c854c68.png)
